framework/generated/vulkan_generators/vulkan_struct_decoders_to_string_body_generator.py

```diff
@@ -193,7 +193,7 @@ def makeStructBody(self, name, values):
                 if value.is_array:
```

Comment:
Changes in this file only affect the behaviour with null handles (previously they were output as numeric zeros).